quic/s2n-quic-qns/src/tls.rs

```diff
@@ -2,9 +2,61 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use crate::Result;
-use std::{path::PathBuf, str::FromStr};
+use s2n_quic::provider::tls::s2n_tls::{keylog::KeyLog, ConfigLoader, ConnectionContext};
+use std::{path::PathBuf, str::FromStr, sync::Arc, time::SystemTime};
 use structopt::StructOpt;
 
+pub static TICKET_KEY: [u8; 16] = [0; 16];
```

Comment:
Can we generate a random key at boot? or better, have the CLI pass the key? that way we're not promoting hard-coding these everywhere :)